diff --git a/model_efdc_dll/native/efdc_fortran_dll/original_efdc_files/RWQC1.for b/model_efdc_dll/native/efdc_fortran_dll/original_efdc_files/RWQC1.for
index 1fbad036b..3e8b24c1e 100644
--- a/model_efdc_dll/native/efdc_fortran_dll/original_efdc_files/RWQC1.for
+++ b/model_efdc_dll/native/efdc_fortran_dll/original_efdc_files/RWQC1.for
@@ -2205,16 +2205,16 @@ C INITIALIZE
               write(1,'(a,a)') 'VERTICAL VELOCITY, ALGAL-DENSITY,'
      &             ,' SOLAR RADIATION, chl-a PRINT AT EACH LAYER'
               write(1,'(a,i4,a,i4)') 'I=',iww(i),'   J=',jww(i)
-              write(1,7111) '          tm'
-     & ,((('vel_',nsp,'_',k),nsp=1,NXSP),k=KC,1,-1)
-     & ,((('den_',nsp,'_',k),nsp=1,NXSP),k=KC,1,-1)
-     & ,((('sol_',nsp,'_',k),nsp=1,NXSP),k=KC,1,-1)
-     & ,(('chl_',k),k=KC,1,-1)
+c             write(1,7111) '          tm'
+c    & ,((('vel_',nsp,'_',k),nsp=1,NXSP),k=KC,1,-1)
+c    & ,((('den_',nsp,'_',k),nsp=1,NXSP),k=KC,1,-1)
+c    & ,((('sol_',nsp,'_',k),nsp=1,NXSP),k=KC,1,-1)
+c    & ,(('chl_',k),k=KC,1,-1)
               CLOSE(1)
           enddo
         ENDIF
       ENDIF
- 7111 format(a, 3(<kc>(<NXSP>(3x,a,i2.2,a,i2.2))),<kc>(6x,a,i2.2) )
+c 7111 format(a, 3(<kc>(<NXSP>(3x,a,i2.2,a,i2.2))),<kc>(6x,a,i2.2) )
 !{ GeoSR Diatom, Green algae Salinity TOX : jgcho 2019.11.27
       if (IWQDGSTOX.eq.1) then
         IF(MYRANK.EQ.0)THEN
